packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/web-services/run-transaction-endpoint.ts

```diff
@@ -0,0 +1,119 @@
+import { Express, Request, Response } from "express";
```

Comment:
@hxlaf 
```suggestion
import type { Express, Request, Response } from "express";
```

Ultimately, using this syntax will allow us to have `express` as a dev dependency only

Reply:
Thank you! Done.